frontends/election-manager/src/utils/election.ts

```diff
@@ -174,6 +176,37 @@ export function getContestsForPrecinct(
   });
 }
 
+export function generateWriteIn(position: number): WriteInCandidate {
+  return {
+    id: 'write-in',
+    isWriteIn: true,
+    name: 'WRITE-IN',
+    writeInPosition: position,
+  };
+}
```

Comment:
Since this isn't a real write-in candidate and just a prefilled one for test deck generation, I think we should somehow make that distinction clear, e.g. by renaming `generateWriteIn` `generateTestDeckWriteIn`. I realize that's imperfect, though, since calling `generateTestDeckWriteIn` within `candidateAtPosition` could be confusing

Happy to bounce ideas!

Reply:
Thanks for the name switch suggestion!

The resulting could be a little confusing, but not too confusing. And I'm renaming the function `candidateAtPosition` to `getTestDeckCandidateAtIndex` to be maximally specific, although a mouthful. Sound better?

These functions are written specifically for test decks so I agree it's good to name them specifically for test decks too.

Reply:
Nice! I agree that naming specifically for test decks through and through makes sense. And I also think that verbose names are okay for the sake of clarity!